core/src/main/resources/error/error-classes.json

```diff
@@ -160,6 +153,10 @@
     "message" : [ "Unsupported literal type %s %s" ],
     "sqlState" : "0A000"
   },
+  "UNSUPPORTED_MODE" : {
```

Comment:
`mode` is also a parameter of the aes function, shall we reuse `INVALID_PARAMETER_VALUE`

Reply:
Actually, any failure/error can be considered as a result of an invalid parameter. We could consider the entire universe as the input to Spark's code.

Reply:
@cloud-fan What is the use case for the sqlState = 0A000 (feature not supported, see https://github.com/apache/spark/tree/master/core/src/main/resources/error) from your point of view?